com.unity.shadergraph/Editor/Data/Graphs/GraphData.cs

```diff
@@ -2039,6 +2146,20 @@ public override void OnAfterDeserialize(string json)
                         keywordGuidMap[input0.m_Guid.m_GuidSerialized] = keyword;
                     }
 
+                    foreach (var serializedDropdown in graphData0.m_SerializedDropdowns)
```

Comment:
Do we need to handle dropdowns in old master node graphs coming in? Are we backporting dropdowns to master nodes?

Reply:
This is a good question, should definitely get clarity on that flow.

Reply:
If we are not (which I would hope at this point tbh), then all the graphdata0 or abstractmaterialnode0 stuff should be unnecessary

Reply:
Cheers, I have removed this now.